micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/MicronautProcessEngineConfigurationCustomizer.kt

```diff
@@ -0,0 +1,20 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import info.novatec.micronaut.camunda.bpm.feature.DefaultProcessEngineConfigurationCustomizer
+import info.novatec.micronaut.camunda.bpm.feature.ProcessEngineConfigurationCustomizer
+import io.micronaut.context.annotation.Replaces
+import org.camunda.bpm.engine.ProcessEngineConfiguration
+import javax.annotation.Nonnull
+import javax.inject.Singleton
+
+@Singleton
+@Replaces(DefaultProcessEngineConfigurationCustomizer::class)
+class MicronautProcessEngineConfigurationCustomizer : ProcessEngineConfigurationCustomizer {
+    override fun customize(@Nonnull configuration: ProcessEngineConfiguration) {
+        configuration.processEngineName = PROCESS_ENGINE_NAME
+    }
+
+    companion object {
+        const val PROCESS_ENGINE_NAME = "CustomizedEngine"
```

Comment:
If this is just used within this class, I'd prefer to have it outside the class, but within the same file and make it a `private const val` instead. That way you're avoiding the caveat of having a `companion object`.

Reply:
It is referenced in ProcessEngineFactoryTest. So I think it should stay that way, right?

Reply:
Then it should stay this way, yes. I didn't see it without checking out the code ;)